qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/services/completions/QuteCompletionsForExpression.java

```diff
@@ -252,8 +253,7 @@ private void fillCompletionFields(ResolvedJavaTypeInfo resolvedType, Range range
 			// class B {String foo;}
 			if (!existingProperties.contains(fieldName)) {
 				CompletionItem item = new CompletionItem();
-				item.setLabel(fieldName);
-				item.setLabel(fieldName + " : " + field.getType());
+				item.setLabel(fieldName + " : " + getSimpleType(field.getType()));

```

Comment:
At first you can simplify with

`item.setLabel(fieldName + " : " + field.getJavaElementSimpleType());`

but to be consistent with method, I will move this code in getSimpleSignature in JavaFieldInfo.